core/src/main/resources/hudson/model/Messages_fr.properties

```
@@ -91,45 +156,119 @@ Hudson.ReadPermission.Description=\
 Queue.Unknown=???
-
+Queue.FinishedWaiting=Attente terminée
 Queue.WaitingForNextAvailableExecutor=En attente du prochain exécuteur disponible
-Queue.WaitingForNextAvailableExecutorOn=En attente du prochain exécuteur disponible sur {0}
+Queue.WaitingForNextAvailableExecutorOn=En attente du prochain exécuteur disponible sur ‘{0}‘
+Queue.init=Restoration de la file de builds
+Queue.node_has_been_removed_from_configuration={0} a été supprimé de la configuration
+Queue.executor_slot_already_in_use=Executor slot déjà utilisé
+Queue.ExceptionCanTake=Exception lors de l''évaluation d''un nœud disponible pour la tâche
+Queue.ExceptionCanRun=Exception lors de l''évaluation d''un nœud pouvant exécuter la tâche
+
+ResultTrend.Aborted=Annulé
+ResultTrend.Failure=Échec
+ResultTrend.Fixed=Résolu
+ResultTrend.NotBuilt=Non exécuté
+ResultTrend.NowUnstable=Maintenant instable
+ResultTrend.StillFailing=Toujours en échec
+ResultTrend.StillUnstable=Toujours instable
+ResultTrend.Success=Succès
+ResultTrend.Unstable=Instable
+Run._is_waiting_for_a_checkpoint_on_={0} en attente d''un point de contrôle sur {1}
 Run.BuildAborted=Le build a été annulé
-Run.MarkedExplicitly=marqué explicitement pour conservé l''enregistrement
+Run.MarkedExplicitly=marqué explicitement pour conserver l''enregistrement
 Run.Permissions.Title=Historique des builds
+Run.running_as_=Exécution en tant que {0}
+Run.running_as_SYSTEM=Exécution en tant que SYSTEM
 Run.UnableToDelete=Impossible de supprimer {0}: {1}
 Run.DeletePermission.Description=\
   Cette option permet aux utilisateurs de supprimer manuellement des builds spécifiques dans l''historique de build.
 Run.UpdatePermission.Description=\
   Cette option permet aux utilisateurs de mettre à jour la description et d''autres propriétés d''un build, \
   par exemple pour laisser des notes sur la cause d''échec d''un build.
+Run.ArtifactsPermission.Description=\
+  Ce droit permet de récupérer les artefacts produits par les builds. \
+  Si un utilisateur ne doit pas accéders aux artefacts, vous pouvez en désactivant ce droit.
+Run.InProgressDuration={0} et décompte
+Run.NotStartedYet=Pas encore commencé
+Run.ArtifactsBrowserTitle=Artefacts de {0} {1}
+
+Run.Summary.Stable=stable
+Run.Summary.Unstable=instable
+Run.Summary.Aborted=annulé
+Run.Summary.NotBuilt=non construit
+Run.Summary.BackToNormal=de retour à la normale
+Run.Summary.BrokenForALongTime=en échec depuis longtemps
+Run.Summary.BrokenSinceThisBuild=en échec depuis ce build
+Run.Summary.BrokenSince=en échec depuis le build {0}
+Run.Summary.Unknown=?
 
+Slave.InvalidConfig.Executors=Configuration invalide de l''agent {0}. Nombre invalide d''exécuteurs.
+Slave.InvalidConfig.NoName=Configuration invalider de l''agent. Le nom est vide
+Slave.Network.Mounted.File.System.Warning=Êtes-vous sûr de vouloir utiliser un système de fichier monté à travers le réseau pour le FS root? Notez que ce dossier n''a pas besoin d''être visible par le contrôleur.
```

```suggestion
Slave.Network.Mounted.File.System.Warning=Êtes-vous sûr de vouloir utiliser un système de fichier monté à travers le réseau pour le système de fichier root? Notez que ce dossier n''a pas besoin d''être visible par le contrôleur.
```
Not sure about this one.